airbyte-integrations/connectors/source-amazon-seller-partner/source_amazon_seller_partner/streams.py

```diff
@@ -275,8 +275,9 @@ def report_options(self) -> Optional[Mapping[str, Any]]:
     def stream_slices(
         self, sync_mode: SyncMode, cursor_field: List[str] = None, stream_state: Mapping[str, Any] = None
     ) -> Iterable[Optional[Mapping[str, Any]]]:
-        start_date = max(pendulum.parse(self._replication_start_date), pendulum.now("utc").subtract(days=90))
-        end_date = pendulum.now("utc")
+        now = pendulum.now("utc")
+        start_date = max(pendulum.parse(self._replication_start_date), now.subtract(days=90))
```

Comment:
should we have `now.subtract(days=90)` declared on the stream instance level for more flexible configuration?

Reply:
I'm not sure since this is the default report time window

Reply:
This part has now been removed because (I believe) we should start syncing the data from the start date provided, but use slices to split requests by applicable date ranges.